### PR TITLE
New UI for app settings

### PR DESCRIPTION
**Pull request by eightants**
## Summary

From the Figma for this issue:
https://github.com/highlight/highlight/issues/4917

Moved the ToggleRow to a separate reusable component

## How did you test this change?

Screenshot 2023-05-16 at 3 24 17 PM

### CODE REVIEW

File: packages/ui/vite.config.ts.timestamp-1683841674780.mjs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// vite.config.ts
```
Comment (et) [inline packages/ui/vite.config.ts.timestamp-1683841674780.mjs:1]: How did this file make it here? Might be a reflame issue?

File: frontend/src/components/ToggleRow/ToggleRow.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { Box, Stack, Text } from '@highlight-run/ui'
+
+import Switch from '@/components/Switch/Switch'
+
+export const ToggleRow = (
+	label: string,
+	info: string | undefined,
```
Comment (et) [inline frontend/src/components/ToggleRow/ToggleRow.tsx:7]: Realize you're just moving this component but can `info` be required? i.e. can we remove `| undefined`?

**Review by et — APPROVED** [review]
Looks good! There is a build failure. Let me know if you need any help diagnosing it.

File: packages/ui/vite.config.ts.timestamp-1683841674780.mjs
Comment (Vadman97) [inline packages/ui/vite.config.ts.timestamp-1683841674780.mjs:1]: this is some weird vite actifact where if the build crashes, the file is left over. should delete it and commit the deletion

File: packages/ui/vite.config.ts.timestamp-1683841674780.mjs (hunk 1 above)
Comment (Vadman97) [reply packages/ui/vite.config.ts.timestamp-1683841674780.mjs:1]: this is some weird vite actifact where if the build crashes, the file is left over. should delete it and commit the deletion

**Review by Vadman97 — APPROVED** [review]

**Pull request merged**